dist/common/systemd/scylla-housekeeping-daily.service

```diff
@@ -3,10 +3,12 @@ Description=Scylla Housekeeping daily mode
 After=network.target
 
 [Service]
+PermissionsStartOnly=true
 Type=simple
 User=scylla
 Group=scylla
 EnvironmentFile=/etc/sysconfig/scylla-housekeeping
+ExecStartPre=/bin/bash -c '/usr/bin/chmod 644 $REPO_FILES'
```

Comment:
It's wrong to play with permissions like this in a .service file. The user ends up with permissions changed behind their back and there's no way to understand what changed it.

It's the user's responsibility to set correct permissions, and our responsibility is to provide good errors, not to fix problems in the background. I suggest to fix s-c-t instead. We can improve the error messages if they're not clear.